src/test/java/org/mockitousage/constructor/CreatingMocksWithConstructorTest.java

```diff
@@ -83,7 +92,7 @@ public void mocking_inner_classes_with_wrong_outer_instance() {
             fail();
         } catch (MockitoException e) {
             assertThat(e).hasMessage("Unable to create mock instance of type 'InnerClass'");
-            assertThat(e.getCause()).hasMessageContaining("Unable to find a matching 1-arg constructor for the outer instance.");
+            assertThat(e.getCause()).hasMessageContaining("Please ensure that it has a constructor that matches these arguments: [foo]");
```

Comment:
I think it is more useful to the user to see the types of arguments rather than the value of the argument.

If we want to push this further, we can even inspect the existing constructor arguments for the class and list them out to the user. I don't find it necessary to merge this change, though. Food for thought.